testing/streaming-e2e/bot/index.js

```diff
@@ -119,8 +130,8 @@ server.post('/api/token/directlinease', async (req, res) => {
         } else {
             console.log(
                 `Requesting Direct Line ASE token using secret "${DIRECT_LINE_SECRET.substr(
-                0,
-                3
+                    0,
+                    3
                 )}...${DIRECT_LINE_SECRET.substr(-3)}"`
```

Comment:
Maybe either `DIRECT_LINE_SECRET?.substr(-3)` or do an argument check (if not already done elsewhere)?

Reply:
I wanted to keep this change scoped to just Cloud Adapter. As this is code for CI, I think it's okay to expect the environment is set up correctly.